### PR TITLE
Fixing prefixless alt coins in getProtfolio (kraken)

### PR DESCRIPTION
**Pull request by cmroche**
This addresses an issue where getProtfolio was unable to check the result for pre-fixless currency pairs due to an accidental "addPrefix" call that was not removed when support for these alt coins was added.

### CODE REVIEW

**Comment by cmroche** [comment]
This will fix issue #1328 and should be perfectly safe for prefixed common assets, please check the notes on that issue for more details.

**Comment by cmroche** [comment]
Bonus bug fix, fees were incorrect

**Comment by askmike** [comment]
@cmroche great stuff, as usual, thanks :)

File: exchanges/kraken.js [hunk 1]
```diff
@@ -203,8 +203,12 @@ Trader.prototype.getPortfolio = function(callback) {
   this.kraken.api('Balance', {}, _.bind(setBalance, this));
 };
 
+// This assumes that only limit orders are being placed with standard assets pairs
+// It does not take into account volume discounts.
+// Base maker fee is 0.16%, taker fee is 0.26%.
```
Comment (askmike) [inline exchanges/kraken.js:208]: Gekko should always be making (unless the price moves hard between a ticker fetch and order placement, which on Kraken I would not rule out).

**Pull request merged**